### ISSUE TITLE
Allow "long hands" when using the wand

### ISSUE THREAD

**Issue by madmaxoft** [cuberite/WorldEdit#15]
When using the wand, make the player capable of reaching far by using a `cLineBlockTracer` to find the actual block that has been clicked (just like a `//hpos` command). This will allow the player to select blocks without having to move towards them.

**Comment by NiLSPACE** [comment]
Bukkit has `/farwand` for that. It isn't that hard to implement. I'll do this.

**Comment by madmaxoft** [comment]
I don't think we need a special type of wand for that, I don't see a reason not to use the normal wand.

**Comment by NiLSPACE** [comment]
Wel, that's how the default bukkit does it, so we should probably at least implement the `/farwand` command.
